### PR TITLE
revert(Dockerfile): RHINENG-12526 go back to UBI9 as the base image

### PR DESCRIPTION
**Pull request by skateman**
## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [x] General Coding Practices

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -20,25 +20,22 @@ WORKDIR /opt/app-root/src
 COPY ./.gemrc.prod /etc/gemrc
 COPY ./Gemfile.lock ./Gemfile /opt/app-root/src/
 
-RUN FULL_RHEL=$(microdnf repolist --enabled | grep rhel-8);                                   \
```
Comment (skateman) [inline Dockerfile:23]: @MichaelMraka I never understood the purpose of the `FULL_RHEL` variable here. I think Marley copied the snippet from your Dockerfile without context. Is there any case when the build is running on a full RHEL machine?

Comment (MichaelMraka) [reply Dockerfile:23]: Yes, when the image is built on internal jenkins.
Anyway we've already stop using it and I think you can remove it as well.

**Review by romanblanco — APPROVED** [review]
LGTM :+1:

**Pull request merged**